### DISCUSSION TITLE
Is it possible to overide the ohmyfetch version in package json?

### DISCUSSION THREAD

**Discussion by vanling** [nuxt/nuxt#18093] [Questions] [answered]
I (*dramatize*) desperately need formdata support in nuxt3. It's fixed recently in https://github.com/nuxt/framework/pull/2379

I could for the time being just apply that small fix myself, but for the future me.. Would there be an easy way to override that specific package version in nuxt3 using the package.json?

I think its part of nuxt3 / nitro?

**Comment by danielroe** [comment, marked as answer]
Largely this depends on your package manager. Often just adding an explicit dependency is enough. You can also implement with `resolutions` (yarn) or `overrides` (npm 8):

```json
{
  "resolutions": {
    "ohmyfetch": "^0.4.11"
  }
}

**Reply by vanling** [reply]
Ah thank you. looking into NPM 8 overrides now.

I think i have it working with the following in the package.json
```
"overrides": {
      "ohmyfetch": "^0.4.11"
 }
```
